app/components/ui/ListingsNavigationLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { SiteSearchInput } from "components/ui/SiteSearchInput";
+import { useMediaQuery } from "react-responsive";
+import styles from "./ListingsSecondaryNavigationInnerLayout.module.scss";
+
+/**
+ * Renders mobile and desktop views for Listings navigation
+ *
+ * @param param0
+ * @returns
+ */
+export const ListingsNavigationLayout = () => {
+  const isMobile = useMediaQuery({ query: "(max-width: 767px)" });
+
+  if (isMobile) {
+    return (
+      <div className={styles.mobileInnerContainer}>
+        <div className={styles.mobileInnerContainerLeft}>
+          <h2>Services</h2>
+          <details className={styles.searchInputDisclose}>
+            <summary>
+              <button
+                type="button"
+                aria-label="Show and hide search input button"
+                className={`${styles.mobileSearchButton} fa-solid fa-magnifying-glass`}
+              />
+            </summary>
+            <SiteSearchInput />
+          </details>
```

Comment:
Few a11y notes:
1. If the button shows the search input, then there should be a state on the aria-label that says whether the button is showing or hiding the input. 
2. It's redundant to use the word "button" in an aria-label because it can cause confusion i.e. is this showing the `search input` or the `search input button`?
3. Clicking the button should shift the focus state to the input automatically